R/hgraph.R

```diff
@@ -195,7 +195,10 @@ hGraph <- function(
     names(md) <- seq_len(nrow(m))
     md$from <- seq_len(nrow(md))
     md <- reshape(data = md, varying = seq_len(nrow(m)), v.names = "w", idvar = "from", timevar = "to", direction = "long")
-    md <- md[md$w > 0, , drop = FALSE]
+    md <- md[(md$w > 0) & (md$w <= 1), , drop = FALSE]
+    if(any(md$w < 0) | any(md$w > 1)){
```

Comment:
Do you mean `w <= 0`? To be a perfect complement set to the condition above.

Reply:
Thanks Nan, I agree with the complement logic per programming practice.  But I think the popup message here is to remind user to be cautious to set **negative** weight. Commonly the weight matrix `m` could contains several `0`'s to indicate no transition relationship between each other, so we do not need to popup this message if the matrix contains `0`.